content/shared/influxdb3-get-started/setup.md

```diff
@@ -479,6 +480,31 @@ influxdb3 show databases --token YOUR_AUTH_TOKEN
 ```
 {{% /code-placeholders %}}
 
+{{% /tab-content %}}
+{{% tab-content %}}
+
+Run the CLI in a Docker container using the `INFLUXDB3_AUTH_TOKEN` environment variable:
+
+{{% code-placeholders "YOUR_AUTH_TOKEN" %}}
+```bash
+docker run --rm \
+  -e INFLUXDB3_AUTH_TOKEN=YOUR_AUTH_TOKEN \
+  quay.io/influxdb/influxdb3:latest \
+  query \
+  --host http://host.docker.internal:8181 \
+  --database example_db \
+  "SHOW TABLES"
+```
+{{% /code-placeholders %}}
+
+> [!Note]
+> Ensure that the database `example_db` exists in the running container.
+> To create it, run:
+>
+> ```bash
+> docker exec -it influxdb3-core influxdb3 create database example_db --token YOUR_AUTH_TOKEN
+> ```
```

Comment:
@MeelahMe We shouldn't add this in Setup--it's a little misleading and too much detail that isn't directly related; it distracts from the basic goal of the page.